### DISCUSSION TITLE
Support Album Embedding

### DISCUSSION THREAD

**Discussion by ua741** [ente-io/ente#1732] [Enhancements]
This feature would allow users to embed photo albums from the photos app into other platforms, websites, or applications. The embedded album should be interactive, displaying the photos and possibly allowing viewers to navigate through them.

**Comment by alxgsv** [comment]
This would be very helpful for my blog!

**Reply by Luciido** [reply]
> This would be very helpful for my blog!

I was looking into this for the exact same reason!!!
I host with infinityfree, which is free, but they have restrictions on uploading pictures

**Comment by paulxb** [comment]
Google have disabled their api for doing this so I am now looking for an alternate to show albums in my Wordpress site. Ente would be great!
https://developers.google.com/photos/support/updates

**Comment by mnvr** [comment]
This feature has been released! https://ente.io/blog/embed-ente-albums/
